packages/core/cms/faststore/content-types.json

```diff
@@ -147,6 +137,48 @@
                           }
                         }
                       }
+                    },
+                    "regionSlider": {
+                      "title": "RegionSlider SlideOver",
+                      "type": "object",
+                      "properties": {
+                        "title": {
+                          "title": "RegionSlider title",
+                          "type": "object",
+                          "properties": {
+                            "setLocation": {
+                              "title": "Set Location title",
+                              "type": "string",
+                              "default": "Set Location"
+                            },
+                            "changeLocation": {
+                              "title": "Change Postal Code title",
+                              "type": "string",
+                              "default": "Change Location"
+                            },
+                            "changePickupPoint": {
+                              "title": "Change Pickup Point location title",
+                              "type": "string",
+                              "default": "Change Store"
+                            }
+                          }
+                        },
+                        "pickupPointChangeApplyButtonLabel": {
+                          "title": "Change Pickup Point apply button label",
+                          "type": "string",
+                          "default": "Update"
+                        },
+                        "pickupPointChooseAriaLabel": {
+                          "title": "Choose Pickup Point aria-label (radio group)",
+                          "type": "string",
+                          "default": "Choose a store"
+                        },
+                        "pickupPointUnavailableInLocationLabel": {
```

Comment:
I preferred the `noPickupPointsAvailableInLocation` heheh because it gives the idea that a list of pickup points is empty, `pickupPointUnavailable` gives the impression that a specific pickup point is unavailable. It makes sense to you?

Reply:
make senses!